chainstate/constraints-value-accumulator/Cargo.toml

```diff
@@ -8,11 +8,12 @@ rust-version.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-common = {path = '../../common'}
-crypto = {path = '../../crypto'}
-pos-accounting = {path = '../../pos-accounting'}
-utils = {path = '../../utils'}
-utxo = {path = '../../utxo'}
+common = { path = "../../common" }
+crypto = { path = "../../crypto" }
+pos-accounting = { path = "../../pos-accounting" }
+randomness = { path = "../../randomness" }
+utils = { path = "../../utils" }
+utxo = { path = "../../utxo" }
```

Comment:
By the way, is there some tool to format `Cargo.toml` files or was this done manually?

Reply:
Nope... it's just the good, old 'find and replace'. Since this PR touches almost every Cargo.toml, I thought I'd fix what I can fix.